### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by fedeCucchiararo**
Hey Manu, I've just added some projects I've done before the bootcamp

### CODE REVIEW

File: staff/fede-gomez/pre-course/chapter2/project/script.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
+/*********PROJECT AIRLINES*********/
```
Comment (vior1992) [inline staff/fede-gomez/pre-course/chapter2/project/script.js:1]: Nice! 👍

Comment (fedeCucchiararo) [reply staff/fede-gomez/pre-course/chapter2/project/script.js:1]: Thanks!!!

File: staff/fede-gomez/pre-course/chapter2/project/script.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+const mainFunction = function () {
+  //Se preguntará por el nombre de usuario y dará la bienvenida.
+  name = prompt('Bienvenido. Cual es tu nombre?');
+  console.log('Bienvenido ' + name + '. Estos son los vuelos programados para hoy:\n\n');
+  //esta variable guardará la cantidad de vuelos con escala:
+  let numberOfFlightsWithScales = 0;
+  //El usuario visualizará todos los vuelos disponibles de una forma amigable:
+  //El vuelo con origen: Barcelona, y destino: Madrid tiene un coste de XXXX€ y no realiza ninguna escala.
+  flights.forEach(function(flight){
+    if(flight.scale === true){
```
Comment (aleixcam) [inline staff/fede-gomez/pre-course/chapter2/project/script.js]: doing that type of logic operations with booleans is a bad practice, better use only
```javascript
 if(flight.scale){
```

Comment (fedeCucchiararo) [reply staff/fede-gomez/pre-course/chapter2/project/script.js]: Indeed. Thanks!

File: staff/fede-gomez/cv/index.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    </section>
+    <section id="skills">
+        <h2>Skills</h2>
+        <ul>
+            <li>Skill 001</li>
+            <li>Skill 002</li>
+        </ul>
+    </section>
+    <footer>
+
```
Comment (joel-molero) [inline staff/fede-gomez/cv/index.html]: Empty section. Is this intended?

**Review by joel-molero — CHANGES_REQUESTED** [review]
Empty tag.

File: staff/fede-gomez/cv/index.html (hunk 3 above)
Comment (fedeCucchiararo) [reply staff/fede-gomez/cv/index.html]: I'll get rid of it. Thanks!

File: staff/fede-gomez/cv/index.html [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+             Fede Gómez CV
+        </h1>
+        <p>Coder en Skylab Coders Academy</p>
+        <address>
+            <p>Federico Gómez Cucchiararo</p>
+            <p>Calle 20, 08210 Barcelona</p>
+            <a href="mailto:fedegomezcucchiararo@gmail">fedegomezcucchiararo@gmail.com</a>
+        </address>
+    </header>
+    <section id="education">
```
Comment (vior1992) [inline staff/fede-gomez/cv/index.html]: Nice work!
You can use ASIDE for write inside the SECTION's

Comment (fedeCucchiararo) [reply staff/fede-gomez/cv/index.html]: That is really helpful. Thanks!!

File: staff/fede-gomez/cv/index.html [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+        </ul>
+    </section>
+    <section id="experience">
+        <h2>Experience</h2>
+        <ul>
+            <li>2015-2018: Operations Support at IKEA Distribution Services GmbH & Co. KG</li>
+            <li>2012-2014: Fulfillment center Associate at Amazon Logistics Germany GmbH</li>
+            </ul>
+    </section>
+    <section id="skills">
```
Comment (gbaste) [inline staff/fede-gomez/cv/index.html]: the id is not necessary, you must remove it!

File: staff/fede-gomez/cv/index.html (hunk 4 above)
Comment (manuelbarzi) [reply staff/fede-gomez/cv/index.html]: that's NOT RIGHT, do not use ASIDE for non-understood reason. please, re-RTFM: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside

**Pull request merged**